src/helpers/validate.ts

```diff
@@ -111,7 +111,7 @@ export default (
                     return false;
                 }
 
-                if ("maxValue" in validation && intArg > validation.maxValue) {
+                if (validation.maxValue && intArg > validation.maxValue) {
```

Comment:
If `maxValue` is zero this check never continues (less likely but still possible)